templates/incl-adventure-tabs.html

```diff
@@ -3,7 +3,7 @@
         <div class="flex flex-row overflow-x-scroll overflow-y-hidden mb-2 px-2 pt-2" id="adventures-buttons">
             {% for adventure in adventures %}
               {% if 'adventures' not in customizations or (customizations['adventures'][adventure.short_name] and level|int in customizations['adventures'][adventure.short_name]) %}
-                <div class="tab {% if loop.index == 1 %}tab-selected{% endif %} z-10 whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="{{adventure.short_name}}">
+                <div class="tab {% if loop.index == 1 %}tab-selected{% else %}adventure{{ loop.index0 }}{% endif %} z-10 whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="{{adventure.short_name}}">
```

Comment:
I don't agree with this approach, as we use a unique identifier as a class. This should be used as the `id` attribute.
Can we create a new issue and fix this? I also think there is a typo in `{{ loop.index0 }}` and this doesn't work as expected at this point.

Reply:
I made an issue so we don't forget: #3765 (cc @hanna1231)